.env-sample

```diff
@@ -10,3 +10,6 @@ REACT_APP_FACEBOOK_APP_SECRET='facebook-app-secret-here''
 
 # The url for social login in our backend
 REACT_APP_SOCIAL_URL='url-here
+
+# The ur for user login
```

Comment:
Please name the API_URL to something more descriptive

Reply:
@hoslack I have corrected that